lib/pbench/test/__init__.py

```diff
@@ -8,27 +8,31 @@
 
 def on_disk_config(
     tmp_path_factory, prefix: str, setup_func: Callable[[Path], Path]
-) -> dict:
-    """Base on-disk configuration recorder.  It returns a dictionary with two
-    items, "tmp" and "cfg_dir", two Path objects for the temporary directory
-    in which the configuration directory is created.
+) -> dict[Path, Path]:
```

Comment:
I think the return type is `dict[str, Path]`.  (Yeah, there are two `Path`'s in the `dict`, but their keys are `str`'s.)

Reply:
Note that `on_disk_config()`'s caller's return types (and one of their callers) have the same problem.